app/controllers/admin_pages_controller.rb

```diff
@@ -23,17 +23,9 @@ def switch_role user
     user.basic? ? user.gold! : user.basic!
   end
 
-  def search_params
-    params.require(:search).permit(User::CREATE_ATTRS)
-  end
-
   private
-  def search
-    @users = User.by_name_like(params[:name])
-                 .by_admin(params[:admin])
-                 .by_type_of(params[:type_of])
-    @users = @users.by_confirmed if params[:actived]
-    @pagy,
-    @users = pagy(@users, items: Settings.digits.size_of_admin_page)
+
+  def get_users_for_admin
+    @pagy, @users = pagy(User.all)
```

Comment:
sao lại load @users trc ở đây
tìm kiếm xong mới phân trang chứ e?

Reply:
dạ tại vì khi vào index admin page thì chưa search, em cho load toàn bộ users lên ạ, tại vì trang index của admin khi vào sẽ là danh sách toàn bộ user trên hệ thống